report/test-assets/faux-psi.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/** @fileoverview This file is a glorified call of prepareLabData. */
+
+/* global document window */
+
+/** @typedef {import('../clients/psi.js').PrepareLabDataResult} PrepareLabDataResult */
+
+(async function __initPsiReports__() {
+  // @ts-expect-error
+  const mobileLHR = window.__LIGHTHOUSE_JSON__;
+  const desktopLHR = JSON.parse(JSON.stringify(mobileLHR));
+
+  const lhrs = {
+    'mobile': mobileLHR,
+    'desktop': desktopLHR,
+  };
+
+  for (const [tabId, lhr] of Object.entries(lhrs)) {
+    await distinguishLHR(lhr, tabId);
+
+    // @ts-expect-error
+    const pldd = /** @type {PrepareLabDataResult} */ (window.prepareLabData(lhr, document));
+    const {scoreGaugeEl, perfCategoryEl,
+      finalScreenshotDataUri, scoreScaleEl, installFeatures} = pldd;
+
+    const container = document.querySelector(`#${tabId} main`);
+    if (!container) throw new Error('Unexpected DOM. Bailing.');
+    container.append(scoreGaugeEl);
+    container.append(scoreScaleEl);
+    if (finalScreenshotDataUri) {
+      const imgEl = document.createElement('img');
+      imgEl.src = finalScreenshotDataUri;
+      container.append(imgEl);
+    }
+    container.append(perfCategoryEl);
+    installFeatures(container);
+  }
+})();
+
+
+/**
+ * Tweak the LHR to make the desktop and mobile reports easier to identify.
+ * Adjusted: Perf category name and score, and emoji placed on top of key screenshots.
+ * @param {LH.Result} lhr
+ * @param {string} tabId
+ */
+async function distinguishLHR(lhr, tabId) {
```

Comment:
can this be combined with what happens in `tweakLhrForPsi`?

Reply:
i could but then i wouldn't have the `single-category` dist-report. but i really want that for local report dev. :)